pkg/smartcontract/manifest/manifest.go

```diff
@@ -135,3 +137,176 @@ func (m *Manifest) DecodeBinary(r *io.BinReader) {
 		r.Err = err
 	}
 }
+
+// ToStackItem converts Manifest to stackitem.Item.
+func (m *Manifest) ToStackItem() (stackitem.Item, error) {
+	groups := make([]stackitem.Item, len(m.Groups))
+	for i := range m.Groups {
+		groups[i] = m.Groups[i].ToStackItem()
+	}
+	supportedStandards := make([]stackitem.Item, len(m.SupportedStandards))
+	for i := range m.SupportedStandards {
+		supportedStandards[i] = stackitem.Make(m.SupportedStandards[i])
+	}
+	abi := m.ABI.ToStackItem()
+	permissions := make([]stackitem.Item, len(m.Permissions))
+	for i := range m.Permissions {
+		permissions[i] = m.Permissions[i].ToStackItem()
+	}
+	trusts := stackitem.Item(stackitem.Null{})
+	if !m.Trusts.IsWildcard() {
+		tItems := make([]stackitem.Item, len(m.Trusts.Value))
+		for i := range m.Trusts.Value {
+			tItems[i] = stackitem.NewByteArray(m.Trusts.Value[i].BytesBE())
+		}
+		trusts = stackitem.Make(tItems)
+	}
+	extra := stackitem.Make("null")
+	if m.Extra != nil {
+		e, err := json.Marshal(m.Extra)
+		if err != nil {
+			return nil, err
+		}
+		extra = stackitem.NewByteArray(e)
+	}
+	return stackitem.NewStruct([]stackitem.Item{
+		stackitem.Make(m.Name),
+		stackitem.Make(groups),
+		stackitem.Make(supportedStandards),
+		abi,
+		stackitem.Make(permissions),
+		trusts,
+		extra,
+	}), nil
+}
+
+// FromStackItem converts stackitem.Item to Manifest.
+func (m *Manifest) FromStackItem(item stackitem.Item) error {
+	str, ok := item.Value().([]stackitem.Item)
```

Comment:
It can be an array, but C# node will fail to unwrap an array, it only works with structs.

Reply:
Fixed everywhere.